### PR TITLE
I1773 globalodoconfig docs

### LINKED ISSUE #1773: Create documentation around GLOBALODOCONFIG env variable
[kind/Documentation]

## What did you find missing in the documentation?
No doc found around ```GLOBALODOCONFIG```

## What is the relevance of it?
Increase odo user experience

### PR DESCRIPTION
**Pull request by boczkowska**
## What is the purpose of this change? What does it change?
Added a note about GLOBAODOCONFIG in CLI

## Was the change discussed in an issue?
fixes #1773

### CODE REVIEW

**Comment by amitkrout** [comment]
```
 [odo]  ✗  Waiting for build to finish [30s]
[odo]  ✗  Failed to create component with name ref-test. Please use `odo config view` to view settings used to create component. Error: unable to wait for build ref-test-app-1 to run: timed out waiting for the condition 
```
/test integration

**Comment by surajnarwade** [comment]
```
level=fatal msg="failed to fetch Terraform Variables: failed to fetch dependency of \"Terraform Variables\": failed to fetch dependency of \"Bootstrap Ignition Config\": failed to fetch dependency of \"Common Manifests\": failed to generate asset \"DNS Config\": getting public zone for \"origin-ci-int-aws.dev.rhcloud.com\": No public route53 zone found matching name \"origin-ci-int-aws.dev.rhcloud.com\""
```
/test integration

**Comment by amitkrout** [comment]
```
 [odo]  ✗  Waiting for component to start [4m]
[odo]  ✗  waited 4m0s but couldn't find running pod matching selector: 'deploymentconfig=nodejs-app' 
```
/test integration

File: docs/cli-reference.adoc [hunk 1]
```diff
@@ -627,12 +627,20 @@ _________________
 
 Modifies odo specific configuration settings within the global preference file.
 
+[NOTE]
+====
+By default the path to the global preference file is `/home/username/.odo/preferece.yaml`, it is stored in the `GLOBALODOCONFIG` variable. You can set up a custom path by changing the value of the variable `GLOBALODOCONFIG="/new_path/preference.yaml"`.
```
Comment (amitkrout) [inline docs/cli-reference.adoc]: ```/home/username/.odo/preferece.yaml``` looks platform specific. Can it be something more generic like ```~/.odo/preferece.yaml```

```it is stored in the `GLOBALODOCONFIG` variable``` -> it is stored in the environment variable `GLOBALODOCONFIG`

You can set up a custom path by changing the value of the variable `GLOBALODOCONFIG="/new_path/preference.yaml"` -> You can set up a custom path by updating the value of the environment variable to  a new preference path, for example `GLOBALODOCONFIG="new_path/preference.yaml"`

I would recommend @mohammedzee1000 to comment here

**Pull request merged**